README.md

```diff
@@ -1,2 +1,213 @@
+
+1. Whether the merger should proceed
+2. What the overall structure of a merged Foundation should look like
+3. How projects should be organized within a merged Foundation
+4. What representation projects should have on a merged Foundation's Board of
+   Directors
+
+To help with the process of organizing, drafting, and submitting the requested
+feedback, the technical projects under both the Node.js and JS Foundations are
+forming a temporary body (the "Node+JS Bootstrap Committee").
+
+This document, along with the other files in this repository, are living
+documents where the Committee will capture and prepare the feedback requested
+by the Boards. The documents here will not represent the finalized governance
+model of a merged Foundation but will serve as input into the generation of
+new Foundation Bylaws and Top Level Project Charters should the merger of the
+Foundations move forward.
+
+Until such time that this group has reached consensus on a new governance
+model, and the Foundation has ratified new By-laws and charters, all projects,
+programs, and committees will continue to operate unchanged.
+
+Upon enactment of a new merged Foundation, or upon decision that the merger
+should not proceed, this working body will be disbanded.
+
+### Contributing to this repository
+
+Anyone may contribute to this repository via pull request. All individuals
+holding commit rights to any Node.js Foundatioon or JS Foundation project may
+participate in the Node+JS Bootstrap Committee and may request commit access
+to this repository.
+
+The [Code of Conduct][] explains the minimum behavior expectations required
+from all contributors.
+
+The Committee follows a [Consensus Seeking][] decision making model. Pull
+Requests must have at least two sign-offs from Committee members and no
+outstanding objections (or "Requests for Changes") in order to land. When there
+is disagreement on any particular issue, Committee members are encouraged to
+talk through the disagreement and come to a compromise resolution.
+
+The Committee does not vote. Should it become impossible to determine consensus
+on a given issue, each of the contending positions should be documented within
+the repository with the arguments for and against each clearly documented for
+further consideration.
+
+### Committee Co-Chairs
+
+Committee Co-Chairs should be selected by vote of the available Committee
+Membership.
+
+### Cocmmittee Meetings
+
+The Node+JS Bootstrap Committee will meet weekly at times selected for best
+possible attendance. Anyone may attend the regular meetings. It will be the
+Chair's responsibility to ensure every meeting has a well-defined agenda posted
+at least 24 hours in advance of the meeting and that meetings are conducted in
+an orderly fashion.
+
+All meetings must be held using Zoom, must be recorded and must be broadcast to
+YouTube. Minute notes must be recorded and published to this repository.
+
+## Guiding Principles
```

Comment:
I have a slight concern that the below copy may reflect an individuals interpretation rather than a consensus across all members. Would it maybe make sense to breaking out the commentary of this section in to a different PR so that we can review it separately from the majority of content here?

Reply:
This is meant as a first iteration. We can tune from here. If there are specific changes, then please suggest them